src/cogent3/app/composable.py

```diff
@@ -1013,7 +1013,10 @@ def _call(self, val, *args, **kwargs):
             return val
 
     type_checked = self._validate_data_type(val)
-    if not type_checked:
```

Comment:
I think there's a cleaner solution to this. Revert your changes in here and change the top conditional of `_validate_data_type` from
```python
def _validate_data_type(self, data):
    """checks data class name matches defined compatible types"""
    if isinstance(data, NotCompleted):
        return data
```
to
```python
def _validate_data_type(self, data):
    """checks data class name matches defined compatible types"""
    if isinstance(data, NotCompleted) and self._skip_not_completed:
        return data

```

Reply:
@GavinHuttley 
I changed this, but it doesn't work. It fails in one test.
Because it doesn't include one case that is NotCompleted and skip is false.

Reply:
@GavinHuttley 
Done